src/Components/Artwork/Metadata.tsx

```diff
@@ -92,6 +103,7 @@ export default createFragmentContainer(Metadata, {
   artwork: graphql`
     fragment Metadata_artwork on Artwork {
       ...Details_artwork
+      internalID
```

Comment:
this change caused all the query updates 😅

Reply:
We could also pass `to` as a prop instead of checking inside of `Metadata` if the component is rendered in the My Collection artwork screen. Like this, we would keep the component more general.